<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormOrdenTrabajoBase</class>
 <widget class="QWidget" name="FormOrdenTrabajoBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>617</width>
    <height>601</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="LNumeroOrden">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;Orden de trabajo #00001&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="textFormat">
      <enum>Qt::RichText</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QToolBox" name="TBGeneral">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="PBasico">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>599</width>
        <height>452</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Datos Basicos</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_17">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_16">
           <item>
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>Numero de Orden de trabajo:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="LNumeroOrdenTrabajo">
             <property name="text">
              <string>TextLabel</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="LFechaIngreso">
             <property name="text">
              <string>Fecha de Ingreso:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDateTimeEdit" name="DTEFechaIngreso">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
             <property name="displayFormat">
              <string>d MMM yyyy HH:mm</string>
             </property>
             <property name="calendarPopup">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="LCliente">
           <property name="text">
            <string>Cliente:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="ECBClientes" name="CBCliente">
           <property name="sizePolicy">
            <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="insertPolicy">
            <enum>QComboBox::NoInsert</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="LTecnico">
           <property name="text">
            <string>Técnico:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="ECBTecnicos" name="CBTecnico">
           <property name="sizePolicy">
            <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="insertPolicy">
            <enum>QComboBox::NoInsert</enum>
           </property>
           <property name="frame">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_18">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_13">
           <item>
            <widget class="QLabel" name="LRequerente">
             <property name="text">
              <string>Requerente:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="LERequerente">
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_14">
           <item>
            <widget class="QLabel" name="LIngresante">
             <property name="text">
              <string>Ingresante:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="LEIngresante">
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_15">
         <item>
          <widget class="QLabel" name="LFechaDevolucion">
           <property name="text">
            <string>Fecha asignada de devolución</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDateEdit" name="DEFechaDevolucion">
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <property name="calendarPopup">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="PEquipamiento">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>599</width>
        <height>452</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Equipamiento</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLabel" name="LEquipamientoDescripcion">
           <property name="text">
            <string>Descripcion:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="LEEquipamientoDescripcion"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QLabel" name="LEquipamientoMarca">
             <property name="text">
              <string>Marca:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="ECBModeloEquipamiento" name="CBEquipamientoMarca">
             <property name="sizePolicy">
              <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QLabel" name="LEquipamientoModelo">
             <property name="text">
              <string>Modelo</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="LEEquipamientoModelo">
             <property name="sizePolicy">
              <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <item>
            <widget class="QLabel" name="LEquipamientoNumeroSerie">
             <property name="text">
              <string>Nº de Serie:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="LEEquipamientoNumeroSerie"/>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_9">
         <item>
          <widget class="QCheckBox" name="CkBEquipamientoGarantia">
           <property name="text">
            <string>En garantía</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="LEquipamientoFactura">
           <property name="text">
            <string>Nº de Factura:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="LEquipamientoFechaFactura">
           <property name="text">
            <string>Fecha de Factura</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_11">
         <item>
          <widget class="QLabel" name="LEquipamientoObservaciones">
           <property name="text">
            <string>Observaciones:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPlainTextEdit" name="PTEEquipamientoObservaciones">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="PHistorial">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>374</width>
        <height>114</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Historial</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QListView" name="LVHistorial"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_12">
         <item>
          <widget class="QPushButton" name="PBAgregarHistorial">
           <property name="text">
            <string>Agregar Accion</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="PBEliminarHistorial">
           <property name="text">
            <string>Eliminar accion</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="PBCerrarOrden">
           <property name="text">
            <string>Cerrar orden</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="PFacturacion">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>254</width>
        <height>114</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Facturacion</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QTableView" name="TVFacturacion"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_10">
         <item>
          <widget class="QPushButton" name="PBAgregarFacturacion">
           <property name="text">
            <string>Agregar</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="PBEliminarFacturacion">
           <property name="text">
            <string>Eliminar</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="PBFacturar">
           <property name="text">
            <string>Facturar</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ECBClientes</class>
   <extends>QComboBox</extends>
   <header>ecbclientes.h</header>
  </customwidget>
  <customwidget>
   <class>ECBModeloEquipamiento</class>
   <extends>QComboBox</extends>
   <header>ecbmodeloequipamiento.h</header>
  </customwidget>
  <customwidget>
   <class>ECBTecnicos</class>
   <extends>QComboBox</extends>
   <header>ecbtecnicos.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
